sgeop/tests/conftest.py

```diff
@@ -0,0 +1,66 @@
+import geopandas.testing
+import numpy
+import pandas
+import pytest
+import shapely
+
+line_collection = (
+    list[shapely.LineString]
+    | tuple[shapely.LineString]
+    | numpy.ndarray
+    | pandas.Series
+    | geopandas.GeoSeries
+)
+
+geometry_collection = (
+    list[shapely.GeometryCollection]
+    | tuple[shapely.GeometryCollection]
+    | numpy.ndarray
+    | pandas.Series
+    | geopandas.GeoSeries
+)
+
+
+def polygonize(
+    collection: line_collection, as_geom: bool = True
+) -> shapely.Polygon | geopandas.GeoSeries:
+    """Testing helper -- Create polygon from collection of lines."""
+    if isinstance(collection, pandas.Series | geopandas.GeoSeries):
+        _poly = geopandas.GeoSeries(collection).polygonize().buffer(0)
+        if as_geom:
+            return _poly.squeeze()
+        else:
+            return _poly
+    else:
+        return shapely.polygonize(collection).buffer(0)
```

Comment:
Same here

Reply:
`shapely.make_valid()` is not fixing the problem locally (perhaps solution in current [dev](https://shapely.readthedocs.io/en/latest/reference/shapely.make_valid.html#shapely-make-valid)) -- still need `.buffer(0)`

Reply:
Without `.buffer(0)` still seeing:

```
FAILED sgeop/tests/test_geometry.py::TestVoronoiSkeleton::test_square[array-True-0.1] - AttributeError: 'GeometryCollection' object has no attribute 'exterior'
FAILED sgeop/tests/test_geometry.py::TestVoronoiSkeleton::test_square_snap_to[list-True-0.001] - AttributeError: 'GeometryCollection' object has no attribute 'exterior'
FAILED sgeop/tests/test_geometry.py::TestVoronoiSkeleton::test_square[list-True-0.001] - AttributeError: 'GeometryCollection' object has no attribute 'exterior'
FAILED sgeop/tests/test_geometry.py::TestVoronoiSkeleton::test_square_snap_to[array-True-0.1] - AttributeError: 'GeometryCollection' object has no attribute 'exterior'
```